### DISCUSSION TITLE
ShowHidden & ZeresPluginLibrary

### DISCUSSION THREAD

**Discussion by Nelna** [JustOptimize/return-ShowHiddenChannels#143] [Q&A] [answered]
Hello !

Today my discord kept crashing, I don't understand why so I removed betterdiscord and put it back and suddenly I had the right to pluggins which no longer work.

Thanks !

[Screenshot](https://pasteboard.co/OvyCTqh9vkdh.png)

**Comment by JustOptimize** [comment, marked as answer]
Check https://github.com/JustOptimize/return-ShowHiddenChannels/issues/140
